BLSQ/iaso
# WC2-53: list dynamic fields

Use entity type list field entries to populate columns and fields popup on beneficiaries list page only ONE entity type is selected

## Self proof reading checklist


## Changes

- adding type selector
- remove unused fields  
- refactor use columns and popup component to use fields from new api
- repair detail infos fields broken due to api change
- allow api to receive multiple entity_type_ids
- add org unit (Health Center) to the hardcoded values in the api

## How to test

Go to entity types page and edit a type with entities.
Add or remove list field items.
Open beneficiaries list page.
Filter those on the types you just edited and check that the columns are correct.
Open map tab and click on a marker to open popup => check that same fields are present .

## Print screen / video

https://user-images.githubusercontent.com/12494624/190381153-1a10bf1b-af1c-4f9a-b53c-4b36cd499534.mov


## Notes

We have some issues with the api, ticket is [here](https://bluesquare.atlassian.net/browse/WC2-59)
